include/aws/mqtt/client.h

```diff
@@ -422,6 +443,23 @@ int aws_mqtt_client_connection_set_reconnect_timeout(
     uint64_t min_timeout,
     uint64_t max_timeout);
 
+/**
+ * Sets the callbacks to call when a connection is success
+ *
+ * \param[in] connection                The connection object
+ * \param[in] on_connection_success     The function to call when a connection succeed
```

Comment:
Trivial "succeeds"

Reply:
"succeeded"?

Reply:
Maybe follow how it's worded above for `on_connection_complete` "The callback to fire when the connection attempt completes successfully" and "The callback to fire when the connection attempt completes unsuccessfully"?